cloud-consumption/src/test/java/com/sequenceiq/consumption/flow/consumption/storage/handler/StorageConsumptionCollectionHandlerTest.java

```diff
@@ -272,6 +299,11 @@ private void mockEnvironmentService(String platform) {
         when(environmentService.getByCrn(ENV_CRN)).thenReturn(detailedEnvironmentResponse);
     }
 
+    private void mockCloudStorageLocationUtil() {
+        doNothing().when(cloudStorageLocationUtil).validateCloudStorageType(FileSystemType.S3, VALID_STORAGE_LOCATION);
```

Comment:
Nit: `doNothing()` is usually not needed because it is the default behavior for `void` return type methods.